src/server/routes/columns.ts

```diff
@@ -56,22 +62,49 @@ export default async (fastify: FastifyInstance) => {
 
   fastify.post<{
     Headers: { pg: string }
-    Body: any
-  }>('/', async (request, reply) => {
-    const connectionString = request.headers.pg
+    Body: PostgresColumnCreate | PostgresColumnCreate[]
+  }>(
+    '/',
+    {
+      schema: {
+        headers: Type.Object({
+          pg: Type.String(),
+        }),
+        body: Type.Union([postgresColumnCreateSchema, Type.Array(postgresColumnCreateSchema)]),
+        response: {
+          200: Type.Union([postgresColumnSchema, Type.Array(postgresColumnSchema)]),
```

Comment:
If we wanted to be really nice about it, would be cool to pluck the table name out of the array, since we don't allow multiple different values for the field here anyway

Reply:
I think it's better to make the response format consistent: `T | T[]` for everything